deploy-jobs-template.yml

```diff
@@ -348,7 +348,7 @@ jobs:
                       },                      
                       {
                         "name": "UseFakedDataSources",
-                        "value": "$(UseFakedDataSources)",
+                        "value": "$(UseFakedDataSourcesInExtract)",
```

Comment:
Suggest this is changed to UseFakedDataSourcesFuncEiirDailyExtract to these variable names a function app based - and so where fakes are used they are either on or off